src/content/docs/faq.mdx

```diff
@@ -66,6 +66,12 @@ import JestMissingGlobals from './shared/jest-missing-globals.mdx'
 
 <JestMissingGlobals />
 
+## `Request`/`Response`/`TextEncoder`/`structuredClone` is not defined (Jest)
```

Comment:
Let's move this under already existing section on the migration page?
https://mswjs.io/docs/migrations/1.x-to-2.x

Reply:
But what is about the people who read the previous FAQ Entry?

Reply:
ah there is the same hint, too ...

https://mswjs.io/docs/migrations/1.x-to-2.x

Reply:
Okay, I created a shared component and imported it to the two locations.